.github/workflows/release.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           distribution: temurin
           java-version: 17.0.6
       - name: Build and publish artifacts
-        run: ./gradlew publishMavenPublicationToSonatypeRepository closeAndReleaseSonatypeStagingRepository -Prelease=true -Pfinal=${{ inputs.release_type == 'final' && 'true' || 'false' }}
+        run: ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository -Pfinal=${{ inputs.release_type == 'final' && 'true' || 'false' }}
```

Comment:
There's just one flag (the `final` one) that I couldn't find a replacement for in the other repos. It essentially tells whether a release version is snapshot or not. When this flag is false or not provided, the version gets `-SNAPSHOT` appended before publishing the artifacts, otherwise the version remains untouched as defined in the `gradle.properties` file.

Having this flag is handy for controlling what type of release we want to publish from within the Gradle script, I couldn't find where in the other repos is the logic that appends (or removes) "-SNAPSHOT" from the version, I'm guessing it might be done in a bash script? @trask @breedx-splk

Reply:
check out https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/version.gradle.kts

this involves setting up the whole branching release workflow that the other repos use

I think it's fine to stay with what you have here for now, and we can discuss sync'ing release workflows in the future